docs/usage/walkthrough.md

```diff
@@ -300,23 +300,23 @@ name                                            url
 p-mysql                                         https://p-mysql.dev.cfdev.sh:443
 p.rabbitmq                                      http://10.144.0.146:8080
 redis-odb                                       http://10.144.0.147:12345
-sm-proxy-a0d0a401-3048-4db7-a6d5-c5098be84d6d   https://cfproxy.dev.cfdev.sh/v1/osb/a0d0a401-3048-4db7-a6d5-c5098be84d6d
+sm-a0d0a401-3048-4db7-a6d5-c5098be84d6d   https://cfproxy.dev.cfdev.sh/v1/osb/a0d0a401-3048-4db7-a6d5-c5098be84d6d
 ```
 
 ### Kubernetes
 
 ```console
-$ kubectl delete clusterservicebrokers sm-proxy-a0d0a401-3048-4db7-a6d5-c5098be84d6d
-clusterservicebroker.servicecatalog.k8s.io "sm-proxy-a0d0a401-3048-4db7-a6d5-c5098be84d6d" deleted
+$ kubectl delete clusterservicebrokers sm-a0d0a401-3048-4db7-a6d5-c5098be84d6d
```

Comment:
I am not sure that this is the case for Kubernetes. 
The Kubernetes brokers are with the original broker name, without prefix and suffix. We may need to adjust this logic in k8s proxy to be the same as in CF proxy.